spectrum/autism_prevalence_map/static/autism_prevalence_map/eslint.config.mjs

```diff
@@ -0,0 +1,12 @@
+import globals from "globals";
+import pluginJs from "@eslint/js";
+import tseslint from "typescript-eslint";
+
+
+/** @type {import('eslint').Linter.Config[]} */
+export default [
+  {files: ["ts/*.{js,mjs,cjs,ts}"]},
+  {languageOptions: { globals: globals.browser }},
+  pluginJs.configs.recommended,
+  ...tseslint.configs.recommended,
```

Comment:
1. use 4 spaces
2. why do we need globals? Why do we need to lint js, mjs, and cjs files in the ts folder? We plan to have these file extensions? Maybe I'm missing something?
3. I think we have to add our own rules, like Chris did for the transmitter, right?

Reply:
I was really just trying to get the seed of something started so that whoever does actual frontend work. I'll get rid of globals, but I don't want to spend forever on this and don't have much time left today

Reply:
How about just copy what Chris did for the transmitter for linting, and we can modify them as we go, instead of adding a new linting config here? That way, we have one method we keep refining, instead of having multiple methods without a clear reason for having multiple.

Reply:
I'm leaving it alone; someone actually working on JS can implement what Chris did for the transmitter. I really just wanted to get something in place that will allow me to push through a commit